src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -117,9 +123,9 @@ String getHostname() {
   }
 
   /**
-   * Checks if a string is a valid host name
+   * Check if the string is a valid host name
    *
-   * @param toVerify a string to be checked against a regular expression
+   * @param toVerify is a string to be checked against a regular expression
```

Comment:
```suggestion
   * @param toVerify a string to be checked against a regular expression
```
You can leave "is" out here because the javadoc tool will create documentation that looks like this: 

**Parameters:**
`toVerify` - a string to be checked against a regular expression